linux_os/guide/system/software/integrity/fips/enable_fips_mode/oval/shared.xml

```diff
@@ -7,19 +7,62 @@
       <extend_definition comment="Dracut FIPS module is enabled" definition_ref="enable_dracut_fips_module" />
       <extend_definition comment="system cryptography policy is configured" definition_ref="configure_crypto_policy" />
       <criterion comment="check if system crypto policy selection in var_system_crypto_policy in the profile is set to FIPS" test_ref="test_system_crypto_policy_value" />
-      {{% if product in ["ol8"] %}}
-      <criterion comment="check if the kernel boot parameter is configured for FIPS mode"
-      test_ref="test_grubenv_fips_mode" />
-      {{% elif product in ["rhel8"] %}}
+      <criterion comment="Check if argument fips=1 for Linux kernel is present in /etc/kernel/cmdline" test_ref="test_fips_1_argument_in_etc_kernel_cmdline" />
```

Comment:
I believe this condition is not applicable for `ol8` and `rhel8`, correct? Should it be moved to the block below?

Reply:
Hello @marcusburghardt  I think it is applicable for RHEL 8, after enabling FIPS mode the file appears there.

Reply:
Ok. Thanks for clarifying @vojtapolasek .